scapy/layers/ikev2.py

```diff
@@ -798,11 +801,8 @@ class IKEv2_payload_CERT_STR(IKEv2_payload_CERT):
 del i, payloadname, name
 IKEv2_class._overload_fields = IKEv2_payload_type_overload.copy()
 
-split_layers(UDP, ISAKMP, sport=500)
-split_layers(UDP, ISAKMP, dport=500)
-
-bind_layers(UDP, IKEv2, dport=500, sport=500)  # TODO: distinguish IKEv1/IKEv2
-bind_layers(UDP, IKEv2, dport=4500, sport=4500)
+# Standard IKE port (IKEv2 and ISAKMP)
+bind_layers(UDP, IKEv2, dport=500, sport=500)
```

Comment:
If I understand it right, the new dissection entry point for IKEv2 is through ISAKMP's dispatch hook.
To make this clearer you could use `bind_top_down` here

Reply:
Good suggestion. I actually flip-flopped between those two variants.

Reply:
addressed in 77de5344977a5f9ab2c73b74d325d9377d77bfee